README.md

```diff
@@ -8,6 +8,10 @@ Opinionated way to create reducers and action creators.
 
   * Unlocks some specifically designed tooling ([eslint-plugin-duck](https://github.com/enkidevs/eslint-plugin-duck)) to speed up the development process by catching as early as possible potential errors
   * Reduce the boilerplate
+  
+Ducks are a modular way to keep your redux action creators and reducers in the same place, simplifying the repo's structure and the development process. Some explanation on why this is beneficial can be found [here](https://github.com/erikras/ducks-modular-redux) and [here](https://medium.freecodecamp.com/scaling-your-redux-app-with-ducks-6115955638be).
+
+Working with ducks means reducers are split into types of data and each reducer (or ~duck~) defines all the actions that are relevant to a specific type of data (e.g. to-do items)
```

Comment:
that's a bit confusing, i think it's mostly a vocabulary issue:
- it's not because you are working with ducks that you can split your reducer, it's pure redux. A duck is just a way to put together your reducer, actions and action creators.
- "types of data"? 
- "each reducer (or ~duck~)", no, a duck is more than just the reducer
- "reducer defines all the actions", no, a reducer is a function taking the previous state and an action as arguments and returning the new state